### PR TITLE
Fix: Optimizing the <SendingInformation/> Component for Tablet Responsiveness

### PR DESCRIPTION
**Pull request by carin91**
This PR is attempting to solve issues 607, 609 and 610.

For the 607 issue the width and min-width properties of the StyledSectionMessage were changed.
For the 609 issue the props of the <StyledSectionMessage/> were adjusted.
For the 610 issue the unnecessary  comments were deleted.

### CODE REVIEW

File: src/components/feedback/SendingInformation/styles.ts [hunk 1]
```diff
@@ -15,8 +15,8 @@ const StyledSectionMessage = styled.div`
       inube.color.surface[appearance!].clear
     );
   }};
-  width: ${(props: IStyledSectionMessageProps) =>
-    props.isMessageResponsive ? "auto" : "100%"};
+  width: 100%;
+  min-width: max-content;
   height: auto;
```
Comment (LinconOssa) [inline src/components/feedback/SendingInformation/styles.ts:20]: You need to use min-width: max-content; height y: auto;

**Review by LinconOssa — CHANGES_REQUESTED** [review]

File: src/components/feedback/SendingInformation/styles.ts (hunk 1 above)
Comment (carin91) [reply src/components/feedback/SendingInformation/styles.ts:20]: Attended with no change.
1) height: auto; is not related to the PR. It should be commented as an issue.
2) min-width: max-content; is required to guarantee to not changed the width of the component under the sized of its children. 

with the min-width property:
image
without the min-width property:
image

**Review by LinconOssa — APPROVED** [review]
meets requirements

**Review by cmarin001 — COMMENTED** [review]
For optimal mobile screen resolution, the layout should resemble the following example: 
image

However, currently, it appears as shown in this image: 
image

As evident from the comparison, there is a misalignment of the close icon in relation to the design specifications provided in the Figma mockups.

**Comment by carin91** [comment]
I think there is a misunderstanding in the review, this is because (1) i included a new feature in the SendingInformationMessage for the <Lines/> and assigned this value in the storybook (2) The SendingInformationMessage uses theme and the storybook uses the -inube- theme, so, it looks different. To see what you want to see i decided to merge this component with main so, you can see what you expected to.

image

image

I want you to notice that the component in surfaces and lines looks exactly the same, you need to assign a value to the new prop to visualize it as in the wireframes.

**Review by cmarin001 — COMMENTED** [review]
The critique pertains not to the color or theme, but to the layout, as observed in the example provided. When configuring the component for responsive design, it becomes evident that there is an issue. The current rendering fails to align with the component specifications as detailed in the Figma design.


This is what I expect to see 
image

however this is what's currently I see
image

**Review by cmarin001 — COMMENTED** [review]
There is a layout error in this component, specifically concerning the close button. Currently, it displays with the same appearance as the component, but it should feature a dark appearance as per the Figma design specifications

image

image

here is what I expect to see 
image

File: src/components/feedback/SendingInformation/index.tsx [hunk 2]
```diff
@@ -129,17 +134,22 @@ const SendInformationMessage = (props: ISendInformationMessageProps) => {
               closeSectionMessage={() => setCancelDisable(false)}
               icon={<IoMdInformationCircleOutline />}
               appearance={appearance}
-              isMessageResponsive={false}
+              buttonType={buttonType}
             >
               <Stack width="100%" gap={inube.spacing.s050}>
                 <Button
```
Comment (cmarin001) [inline src/components/feedback/SendingInformation/index.tsx:140]: Per the Figma design specifications, the spacing for the buttons should be 'compact' when configured for mobile screens. However, at present, this consideration is not being accounted for

image

Comment (carin91) [reply src/components/feedback/SendingInformation/index.tsx:143]: Cristal clear, i adjusted several conditions to guarantee the similitude with the wireframes.

**Pull request merged**